cf-templates/ecs-cluster/config_params.json

```diff
@@ -1,17 +1,18 @@
 {
   "Parameters": {
-    "EcsClusterName": "ecs-workshop-cluster",
+    "EcsClusterName": "pt-ecs-workshop-cluster",
```

Comment:
@mohitkumarsharmaflux7  can you please revert the default names for stack prefix and ECS cluster?